client/blocks/eligibility-warnings/warning-list.tsx

```diff
@@ -40,7 +41,9 @@ export const WarningList = ( { context, translate, warnings, showContact = true
 							</Fragment>
 						) }
 						<span className="eligibility-warnings__message-description">
-							<span>{ description } </span>
+							<span
+								dangerouslySetInnerHTML={ { __html: DOMPurify.sanitize( description ) } } // eslint-disable-line react/no-danger
```

Comment:
In addition to DomPurify, the strings that are received here are all fixed strings that are defined on the server, so security-wise it should be ok.